torch/fx/experimental/symbolic_shapes.py

```diff
@@ -7293,12 +7304,27 @@ def _suggest_torch_checks(
     printer = _PythonMsgPrinter(src_map)
     msg = e.args[0]
     msg += "\nTo fix the error, insert one of the following checks before this call:"
-    # suggested fixes to resolve `cond`` are to tell the compiler to assume
+
```

Comment:
mm would it not be a trivial check on cond?
we do not need to convert it to string and do regex matching ?
we can check cond structure directly. cond is a sympy expression